src/vite/index.ts

```diff
@@ -0,0 +1,92 @@
+import { load as cheerioLoad } from "cheerio";
+import { ftlValuesGlobalName } from "../bin/keycloakify/ftlValuesGlobalName";
+import { generateCssCodeToDefineGlobals, replaceImportsInCssCode } from "../bin/keycloakify/replacers/replaceImportsInCssCode";
+
+// need to be imported from vite/dist/node because of circular dependency error
+import type { RenderBuiltAssetUrl, UserConfig } from "vite/dist/node";
+
+export function withKeycloakify(config: UserConfig = {}): UserConfig {
```

Comment:
What do you think of removing the `withKeycloakify` helper & replacing it with a [config](https://vitejs.dev/guide/api-plugin.html#config) option on the plugin?

Reply:
I thought about this, but we need other changes to the config besides adding the plugin.